assumptions.tex

```diff
@@ -1,9 +1,11 @@
 The assumptions system allows users to specify that symbols have certain
 common mathematical properties, such as being positive, imaginary, or
 integer. SymPy is careful to never perform simplifications on an expression
-unless the assumptions allow them. For instance, the identity $\sqrt{t^2} = t$
-holds if $t$ is nonnegative ($t\ge 0$). However, for general complex $t$, no
-such identity holds.
+unless the assumptions allow them. For instance, the simplification $\sqrt{t^2}
+= t$ holds if $t$ is nonnegative ($t\ge 0$). However, for a complex $t$, the
+simplification is in general invalid, the correct identity that holds for all
```

Comment:
You can write it using `exp`, but that's even more complicated, I just simplified `exp(pi*I)=-1`. See http://www.theoretical-physics.net/dev/math/complex.html#examples